tcms/issuetracker/types.py

```diff
@@ -6,14 +6,15 @@
 """
 
 import os
-from urllib.parse import urlencode
```

Comment:
This actually causes more pylint errors than it fixes. The import order should be stdlib, 3rd party packages, django, then tcms. Add a blank line between the groups of imports to make it easier to visually find things.

There are a couple more issues that you can fix with this file. Either fix them or drop the commit from this PR.